### ISSUE TITLE
[Accessibility] In the editing state, Accessibility Insights is not responding when just select DataGridViewTextBoxCell

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#3925]
* .NET Core Version: .Net SDK 5.0.100-rc.2.20465.23

* Have you experienced this same bug with .NET Framework?: 
No

**More Info:**
This issue cannot reproduce on Inspect and Narrator.

**Minimal repro:**

1. Open the attached [WinFormsApp2.zip](https://github.com/dotnet/winforms/files/5231054/WinFormsApp2.zip) application in VS, then build and run it.
2. Enter text into Column1(eg: winforms).
3. Open Accessibility Insights and focus on the Column1.
4. Observe Accessibility Insights.

**Expected behavior:**
Accessibility Insights is working.
![Issue2steps1](https://user-images.githubusercontent.com/45864649/93316486-d5fcd980-f83e-11ea-8abb-f44ef23dcbab.gif)

**Actual behavior:**
Accessibility Insights is not responding.
![Issue2steps](https://user-images.githubusercontent.com/45864649/93316495-d7c69d00-f83e-11ea-8404-80c0aa37677f.gif)

**Comment by vladimir-krestov** [comment]
The regression was added in .Net Core 3.1 in 4c0522e03b13e6a1c4412486643c148b1fd4ce41 commit.
If to revert changes for DGVTextBox then AccessiblityInsights works well.
It is risky for 5.0 so moved to 6.0.

**Comment by Jessie-Zhang01** [comment]
Verified the issue with latest NET SDK 6.0.100-alpha.1.20557.2, this issue is fixed. Accessibility Insights is working when just select DataGridViewTextBoxCell.
![step3](https://user-images.githubusercontent.com/45864649/98524075-b1a70180-22b1-11eb-95e7-6490142888c4.gif)

**Comment by SergeySmirnov-Akvelon** [comment]
Should be ported to .NET Core 3.1 and .NET Core 5.0

**Comment by SergeySmirnov-Akvelon** [comment]
Has been ported to [3.1](https://github.com/dotnet/winforms/pull/4352) and [5.0](https://github.com/dotnet/winforms/pull/4351)

**Comment by Jessie-Zhang01** [comment]
Verified the issue with latest .NET Core SDK 3.1.406 from February Test Pass, this issue is fixed.
![step1](https://user-images.githubusercontent.com/45864649/105683892-2132a180-5f2f-11eb-8d87-e36c9562293a.gif)

**Comment by Jessie-Zhang01** [comment]
Verified the issue with latest .NET SDK 5.0.103 from February Test Pass, this issue is fixed.
![step2](https://user-images.githubusercontent.com/45864649/105817690-ec871e80-5ff0-11eb-8b78-699e482ba365.gif)
